packages/wonder-stuff-render-environment-jsdom/tsconfig.json

```diff
@@ -7,5 +7,9 @@
     "outDir": "dist",
     "rootDir": "src",
   },
-  "references": []
+  "references": [
+    {"path": "../wonder-stuff-core"},
+    {"path": "../wonder-stuff-server"},
+    {"path": "../wonder-stuff-render-server"},
+  ]
```

Comment:
**suggestion:** you may need to update tsconfig-build.json to include a reference to `wonder-stuff-render-environment-jsdom` so that `.d.ts` files are built for this package.

Reply:
Oh good catch. I've done that here (and for the previous PR - so I won't fix it there).